### PR TITLE
fix: changes in gravity module test files

### PR DESCRIPTION
**Pull request by ruthishvitwit**

### CODE REVIEW

**Review by byte-bandit — DISMISSED** [review]
Some small remarks. Looks good otherwise. Final review to @aleem1314

File: x/gravity/keeper/test_common.go [hunk 1]
```diff
@@ -94,8 +99,8 @@ var (
 			[]govclient.ProposalHandler{
 				paramsclient.ProposalHandler,
 				// distrclient.ProposalHandler,
-				upgradeclient.LegacyProposalHandler,
-				upgradeclient.LegacyCancelProposalHandler,
+				//upgradeclient.LegacyProposalHandler,
+				//upgradeclient.LegacyCancelProposalHandler,
```
[the remark this answers is not in the document]
Comment (deepan95dev) [reply x/gravity/keeper/test_common.go]: It's removed.

**Review by aleem1314 — DISMISSED** [review]

File: x/gravity/keeper/test_common.go [hunk 2]
```diff
@@ -579,10 +591,10 @@ func CreateTestEnv(t *testing.T) TestInput {
 
 			// distribution module balance must be outstanding rewards + community pool in order to pass
 			// invariants checks, therefore we must add any amount we add to the module balance to the fee pool
-			feePool := distKeeper.GetFeePool(ctx)
-			newCoins := feePool.CommunityPool.Add(math.LegacyNewDecCoinsFromCoins(amt...)...)
+			feePool, err := distKeeper.FeePool.Get(ctx)
```
[the remark this answers is not in the document]
Comment (byte-bandit) [reply x/gravity/keeper/test_common.go:594]: `require.NoError(t, err)`

Also remove on line 599, as I don't see it being set again after this call?

**Review by byte-bandit — APPROVED** [review]
Final Review to @aleem1314

**Review by aleem1314 — APPROVED** [review]

**Pull request merged**